src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics.CodeAnalysis;
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-linux`
Basic Job Tests.Wait-Job tests.Can timeout waiting for a job
```powershell
Cannot perform operation because object "Pipeline" has already been disposed.
Object name: 'Pipeline'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Job/Jobs.Tests.ps1: line 139
139:             Get-Job | Where-Object { $_.Id -ne $startedJob.Id } | Remove-Job -ErrorAction SilentlyContinue -Force

```